--- src/starship.toml	2025-11-11 10:16:50
+++ src/starship-no-git.toml	2025-11-11 21:57:55
@@ -26,19 +26,23 @@
 disabled = true
 
 [git_branch]
+disabled = true
 style = "white"
 format = "on [$symbol$branch]($style) "
 
 [git_commit]
+disabled = true
 style = "white"
 
 [git_metrics]
-disabled = false
+disabled = true
 
 [git_state]
+disabled = true
 style = "bold white"
 
 [git_status]
+disabled = true
 style = "white"
 
 [haskell]
